xc/device.ml

```diff
@@ -1493,7 +1493,15 @@ module Dm = struct
 
     let qmp_event_handle domid qmp_event =
       (* This function will be extended to handle qmp events *)
-      debug "Got QMP event, domain-%d: %s" domid qmp_event.event
+      debug "Got QMP event, domain-%d: %s" domid qmp_event.event;
+      qmp_event.data >>= fun d ->
```

Comment:
This is better written as:
```
qmp_event.data >>= function
| RTC_CHANGE offset ->
```

Is this match complete? Can't there be any other values?

Reply:
Updated, thanks.
There will be other values to be added later, currently, we only process RTC_CHANGE